### ISSUE TITLE
AttributeError: 'HTML2FPDF' object has no attribute 'table_offset'

### ISSUE THREAD

**Issue by ruiz-manuel** [PyFPDF/fpdf2#723]
I am getting that error when writing a table from an html code. Originally the html comes from `pandas.DataFrame.Styler.to_html()` but this little code snippet is enough to reproduce the error:

```python
from fpdf import FPDF

pdf = FPDF()

pdf.add_page()

html = """
<table  width=130>
  <tbody>
    <tr>
      <td>1.10</td>
    </tr>
  </tbody>
</table>
"""

pdf.write_html(html)
```

**Comment by Lucas-C** [comment]
Hi @ruiz-manuel

Thank you for reporting this bug!
Thanks to your minimal code I was quickly able to reproduce it.
I'm working on a fix

**Comment by Lucas-C** [comment]
@allcontributors please add @ruiz-manuel for bug

**Comment by ruiz-manuel** [comment]
@Lucas-C awesome, thank you so much!

**Comment by Lucas-C** [comment]
This will be solved by https://github.com/PyFPDF/fpdf2/pull/703

You can test it by yourself:

    pip install git+https://github.com/PyFPDF/fpdf2.git@table

Hence I'm closing this issue.
But feel free to drop a comment @ruiz-manuel if you have any question 😊

**Comment by shindodkar** [comment]
@ruiz-manuel would you help how to this type of projects get solved. Actually i am a beginner and your guidance will be so helpful for me.

**Comment by Lucas-C** [comment]
A fix has been released in v2.7.0: https://github.com/PyFPDF/fpdf2/releases/tag/2.7.0
